networking/getting-traffic-into-cluster/getting-traffic-cluster-ingress-controller.adoc

```diff
@@ -0,0 +1,56 @@
+[id="getting-traffic-cluster-ingress-controller"]
+= Getting traffic into a cluster using an Ingress Controller
+include::modules/common-attributes.adoc[]
+:context: getting-traffic-cluster-ingress-controller
+
+toc::[]
+
+
+{product-title} provides methods for communicating from
+outside the cluster with services running in the cluster. This method uses a
+service external IP.
```

Comment:
Copy-and-paste error: "a service external ip"→"an ingress controller".

Reply:
Crumbs. Thank you, fixing now.